distribution/docs/src/main/resources/content/_integrating/_endpoints/catalog-rest-endpoint.adoc

```diff
@@ -3,291 +3,329 @@
 
-`<metacardId>` is the `Metacard.ID` of the metacard to be updated and `<input transformer>` is the name of the transformer to use when parsing an override metadata attribute (optional).
+.Create Request URL
+[source,javascript]
+----
+https://<FQDN>:<PORT>/services/catalog/
+----
 
-|{url}<metacardId>?transform=<input transformer>
+.Example Create Request
+[source,http,linenums]
+----
+POST /services/catalog?transform=xml HTTP/1.1
+Host: <FQDN>:<PORT>
+Content-Type: multipart/form-data; boundary=----WebKitFormBoundary7MA4YWxkTrZu0gW
+Cache-Control: no-cache
 
-|delete
-// add a link: https://github.com/codice/ddf/issues/4374
-|DELETE
-|The ID of the Metacard to be deleted is appended to the end of the URL.
+------WebKitFormBoundary7MA4YWxkTrZu0gW
+Content-Disposition: form-data; name="parse.resource"; filename=""
+Content-Type:
 
-`<metacardId>` is the `Metacard.ID` of the metacard to be deleted.
 
-|{url}<metacardId>
+------WebKitFormBoundary7MA4YWxkTrZu0gW
+Content-Disposition: form-data; name="parse.metadata"; filename=""
+Content-Type:
 
-|<<{integrating-prefix}catalog_rest_read_example,read>>
-// add a link: https://github.com/codice/ddf/issues/4374
-|GET
-|The ID of the Metacard to be retrieved is appended to the end of the URL.
 
-By default, the response body will include the XML representation of the Metacard.
+------WebKitFormBoundary7MA4YWxkTrZu0gW--
+----
 
-`<metacardId>` is the `Metacard.ID` of the metacard to be retrieved.
+The `create` and `update` methods both support the multipart mime format.
+If only a single attachment exists, it will be interpreted as a resource to be parsed, which will result in a metacard and resource being stored in the system.
 
-|{url}<metacardId>
+If multiple attachments exist, then the REST endpoint will assume that one attachment is the actual resource (attachment should be named `parse.resource`) and the other attachments are overrides of metacard attributes (attachment names should follow metacard attribute names).
+In the case of the metadata attribute, it is possible to also have the system transform that metadata and use the results of that to override the metacard that would be generated from the resource (attachment should be named `parse.metadata`).
 
-|federated read
-// add a link: https://github.com/codice/ddf/issues/4374
-|GET
-|The SOURCE ID of a federated source is appended to the URL before the ID of the Metacard to be retrieved is appended to the end.
+.Create Success
+If the ingest is successful, a status of `201 Created` will be returned, along with the Metacard ID in the header of the response.
 
-`<sourceId>` is the `FEDERATED SOURCE ID` and `<metacardId>` is the `Metacard.ID` of the Metacard to be retrieved.
+.Request with Non-XML Data
+[NOTE]
+====
+If a request with non-XML data is sent to the Catalog REST endpoint,
+the metacard will be created but the content will stored in the `metadata` field.
+This could affect the discoverability of the resource.
+====
```

Comment:
❓ What does `content` mean in this context? The resource?

> the metacard will be created but the content will **be** stored in the `metadata` field.

Also, 🐝 sure to add the "**be**" in the sentence above 😏.

Reply:
fixed and fixed.